### ISSUE TITLE
swig build break

### ISSUE THREAD

**Issue by linas** [opencog/link-grammar#1226]
I get this error:
```
Making all in python
make[2]: Entering directory 'link-grammar-git/build/bindings/python'
  GEN      lg_python_wrap.cc
../../../bindings/swig/link_grammar.i:53: Error: Syntax error in input(1).
make[2]: *** [Makefile:924: lg_python_wrap.cc] Error 1
```

commenting out line 53
```
%delobject destroy_Parse_Options&;
```

works.

I have swig version 3.0.12-2 (debian stable aka debian buster)

**Comment by ampli** [comment]
I use swig 4.0.1. I guess such a definition is not supported on swig 3. This is bad, as it would cause a `ParseOption` memory leak whenever it gets out of scope.
For an immediate fix, I will submit a patch that checks for the `swig` version and skips this line for version <4.
The real fix would be to find an alternate way for defining the needed %delobject. I will investigate that.

**Comment by linas** [comment]
Fixed by #1228
